Makefile

```diff
@@ -14,6 +14,9 @@ test:
 run:
 	sam local invoke NotifySlackFunction -e snsEvent.json
 
+deploy:
+	aws s3 cp bin/*.zip s3://telia-oss/aws-notify-slack/main.zip --acl "public-read"
```

Comment:
Should probably try with just a trailing slash:
`s3://telia-oss/aws-notify-slack/`

At least if you want to continue using a `*` in `bin/*.zip`? Also, might want to use `TRAVIS_TAG` as part of the name so you can version it along with the module?

Reply:
How weird is that on Travis seems to pass and deployment is working 
https://travis-ci.org/telia-oss/aws-notify-slack/builds

Reply:
Yeah, it will keep working as long as you only produce one binary - but then you don't really need the `*` right? 😄